src/js/themes/hpe.js

```diff
@@ -201,6 +201,18 @@ export const hpe = deepFreeze({
       textDecoration: 'underline',
     },
   },
+  avatar: {
+    avatar: {
```

Comment:
is this really how theme is structured? `avatar.avatar`?

Reply:
fixed